<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimeReport::ConnectionDialog</class>
 <widget class="QDialog" name="LimeReport::ConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lrdatabrowser.qrc">
    <normaloff>:/databrowser/images/database_disconnected</normaloff>:/databrowser/images/database_disconnected</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbConnection">
       <property name="text">
        <string>Connection Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leConnectionName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbbUseDefaultConnection">
     <property name="text">
      <string>Use default application connection</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbDriver">
         <property name="text">
          <string>Driver</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbServer">
         <property name="text">
          <string>Server </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbUser">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbPassword">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbDatabase">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cbbDrivers"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leServerName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePort"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leUserName">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lePassword">
           <property name="text">
            <string/>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="lrdatabrowser.qrc">
             <normaloff>:/databrowser/images/closed_eye.png</normaloff>
             <normalon>:/databrowser/images/eye.png</normalon>:/databrowser/images/closed_eye.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QLineEdit" name="leDataBase">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAutoConnect">
     <property name="text">
      <string>Auto connect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbbKeepCredentials">
     <property name="text">
      <string>Don't keep credentials in lrxml</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbCheckConnection">
       <property name="text">
        <string>Check connection</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leConnectionName</tabstop>
  <tabstop>cbbUseDefaultConnection</tabstop>
  <tabstop>cbbDrivers</tabstop>
  <tabstop>leServerName</tabstop>
  <tabstop>lePort</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leDataBase</tabstop>
  <tabstop>cbAutoConnect</tabstop>
  <tabstop>cbbKeepCredentials</tabstop>
  <tabstop>pbCheckConnection</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>pbOk</tabstop>
 </tabstops>
 <resources>
  <include location="lrdatabrowser.qrc"/>
  <include location="lrdatabrowser.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>LimeReport::ConnectionDialog</receiver>
   <slot>slotAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>LimeReport::ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCheckConnection</sender>
   <signal>clicked()</signal>
   <receiver>LimeReport::ConnectionDialog</receiver>
   <slot>slotCheckConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotAccept()</slot>
  <slot>slotCheckConnection()</slot>
 </slots>
</ui>
